tests/ml_commons/test_ml_commons_client.py

```diff
@@ -70,6 +74,47 @@ def test_init():
     assert type(ml_client._model_uploader) == ModelUploader
 
 
+def test_integration_pretrained_model_upload_unload_delete():
+    raised = False
+    try:
+        model_id = ml_client.upload_pretrained_model(
+            model_name=PRETRAINED_MODEL_NAME,
+            model_version=PRETRAINED_MODEL_VERSION,
+            model_format=PRETRAINED_MODEL_FORMAT,
+        )
+    except:
+        raised = True
+    assert raised == False, "Raised Exception during pretrained model upload"
+
+    if model_id:
+        raised = False
+        try:
+            ml_model_status = ml_client.get_model_info(model_id)
+            assert ml_model_status.get("model_format") == "TORCH_SCRIPT"
+            assert ml_model_status.get("algorithm") == "TEXT_EMBEDDING"
+        except:
+            raised = True
+        assert raised == False, "Raised Exception in getting pretrained model info"
+
+        raised = False
+        try:
+            ml_client.unload_model(model_id)
+            time.sleep(60)
+            ml_model_status = ml_client.get_model_info(model_id)
+            print("ml_model_status", ml_model_status)
+            assert ml_model_status.get("model_state") == "UNLOADED"
+        except:
```

Comment:
Lint is failing for bare except, you need to add to skip linting error `# noqa: E722`